<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wrapper_widget</class>
 <widget class="QWidget" name="wrapper_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="main_widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>670</width>
     <height>451</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="wrapper_layout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="logotype_layout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="gsreg_logotype">
         <property name="text">
          <string>GSReg</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="julia_logotype">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="line_layout">
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="Line" name="header_line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="progress_bar_layout">
       <item>
        <widget class="QProgressBar" name="progress_bar">
         <property name="value">
          <number>24</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="title_layout">
       <item>
        <widget class="QLabel" name="step_label">
         <property name="text">
          <string>Step 0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="text">
          <string>TÃ­tulo </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="main_layout"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
